src/components/FlowbiteIllustration.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+
+/**
+ * FlowbiteIllustration - Secure SVG illustration component
+ * 
+ * Features:
+ * - Path traversal protection (sanitizes basePath and name)
+ * - Graceful error handling (collapses to 0 height on load failure) and displays nothing
+ * - Maintains aspect ratio when height is specified
+ * 
+ * Props:
+ * @param {string} name - SVG filename (required, alphanumeric/hyphens/underscores only)
```

Comment:
there should be a set of the names (which can be extended) that devs can use to display particular flowbite illustration (so the svg files should have appropriate names)

e.g. `access`, `error`, `unauthorized`, ....